app/services/id_token_builder.rb

```diff
@@ -52,6 +52,7 @@ def acr
     when 0 then Saml::Idp::Constants::IALMAX_AUTHN_CONTEXT_CLASSREF
     when 1 then Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF
     when 2 then Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF
+    when 22 then Saml::Idp::Constants::IAL2_STRICT_AUTHN_CONTEXT_CLASSREF
```

Comment:
Can we make this `Identity:: IAL2_STRICT` please? That's what I meant about the magic constant

Reply:
Yep.  Missed this one